GOVERNANCE.md

```diff
@@ -13,27 +13,42 @@ The following code repositories are governed by the Cloud Native Buildpacks comm
 * [resources](https://github.com/buildpacks/resources)
 * [community](https://github.com/buildpacks/community)
 * [samples](https://github.com/buildpacks/samples)
+* [github-actions](https://github.com/buildpacks/github-actions)
+* [registry-api](https://github.com/buildpacks/registry-api)
```

Comment:
```suggestion
* [registry-api](https://github.com/buildpacks/registry-api)
* [registry-index](https://github.com/buildpacks/registry-index)
* [regsitry-namespaces](https://github.com/buildpacks/registry-namespaces)
* [libcnb](https://github.com/buildpacks/libcnb)
* [pack-orb](https://github.com/buildpacks/pack-orb)
* [homebrew-tap](https://github.com/buildpacks/homebrew-tap)
* [artwork](https://github.com/buildpacks/artwork)
* [https://github.com/buildpacks/tekton-integration](https://github.com/buildpacks/tekton-integration)
* [ci](https://github.com/buildpacks/ci)
```

But maybe we don't want to maintain a full list? I didn't know about some of these like CI  until I looked them up.

Reply:
I'm fine adding those, but it will probably be difficult to keep up with. I think we're wary of just saying "everything". Probably better to list, and try to keep them up to date

Reply:
Went with everything after discussion with Joe because I despair that we can maintain the master list.